### ISSUE TITLE
"Open Tagged Random Note: opens a random note that has a selected tag." How to select tag?

### ISSUE THREAD

**Issue by kmaustral** [erichalldev/obsidian-smart-random-note#15]
I seem to recall it was possible to select the tag for a random note from a dropdown list. This doesn't seem to be available anymore.

**Comment by erichalldev** [comment]
@kmaustral, will you tell me which version of Obsidian you're using, as well as which version of Smart Random Note?

**Comment by kmaustral** [comment]
It's Obsidian 0.12.9 and Smart Random Note 0.1.3.

Let me know if you need any more information.

Thanks.

**Comment by erichalldev** [comment]
In the command palette there should be an command for "Smart Random Note: Open Tagged Random Note". Is that command available? Alternatively, you can search for a tag in the core search plugin with `tag:#{{tagtosearchfor}}`, then use the "Smart Random Note: Open Random Note from Search" command.

**Comment by kmaustral** [comment]
Yes, that command is available, but it doesn't seem to work. Thanks for the
workaround.
